docs/dev/shop-setup/GitLab_Free.md

```diff
@@ -0,0 +1,52 @@
+# GitLab Free Shop Setup:
+
+### Setting up groups and user permissions:
+
+![GitLab Standalone Group Users & Permissions Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitLab_Standalone_Group_Setup_Users_And_Permissions.png)
+
+#### Trustee:
+
+1. Add the shop manager to the LFP group with maintainer permissions
+
+2. Add the shop developers to the LFP group with reporter permissions
+
+#### Shop Manager:
+
+1. Create a GitLab group for the shop
+
+2. Add the shop developers to the shop group with developer permissions
+
+### Setting up the shop issue board:
+
+![GitLab Shop Board Diagram](https://raw.githubusercontent.com/LibreFoodPantry/ProjectTemplate/shop_setup_documentation/docs/dev/shop-setup/diagrams/GitLab_Setup_Shop_Board.png)
+
+#### Shop Manager:
+
+1. Go to the Development issue board in the shop group
```

Comment:
Why do shop members need reporter permissions in LFP if the board is in the shop group?

* Should the shop board be located in LFP?
* Or, should we not give reporter permissions to shop member in LFP?
* Or, is there another reason shop members need reporter permissions?

I have two motivations with these questions: first, I would rather the shop manager deal with permissions for their shop members rather than a trustee so I'm hoping that members don't need reporter since trustees have to grant them; second, I would like to obey the principle of least privilege.

Reply:
In the original workflow diagram from the June retreat the shop developers were labeled as having reporter permissions in the LFP group, I think that's where I got this from. 
![image](https://user-images.githubusercontent.com/47544822/62243303-07041b80-b3ab-11e9-887b-049f7c365255.png)

I am unsure of where we stand on what boards look like across the different groups and that's where this issue came from: https://github.com/LibreFoodPantry/Community/issues/52

Are we going to have one board in the LFP group where all of the management happens and different shops filter by labels as you suggested here? https://github.com/LibreFoodPantry/Community/issues/37#issuecomment-510214380

Or, is there going to be a board in every shop group that the shop manager handles and pulls issues from the story map or from a board in the LFP group?

The way the documentation is setup now, each shop group has its own board that the shop manager puts issues on. If that's the case, then the shop developers don't need reporter permissions in the LFP group and they don't need be added to the LFP group, just the shop managers.

Reply:
You're right. There is still some murkiness here. I've tried to make some clarifications on #10. I'm not going to hold up this pull-request for these issues.

So I think this "issue" is resolved :)